### DISCUSSION TITLE
Ubuntu version on google VM is incorrect - but it isn't

### DISCUSSION THREAD

**Discussion by djhurlburt** [NightscoutFoundation/xDrip#3478] [Q&A]
The virtual machine instance creation page (https://navid200.github.io/xDrip/docs/Nightscout/VirtualMachine.html) is a bit outdated from the view/options Google now has, however, I'm 99.9% confident I set up the VM properly

e2-micro
x86
us-central-a
http/https traffic on
standard persistent disk

but when I do the nightscout install script I get:
![image](https://github.com/NightscoutFoundation/xDrip/assets/122819185/f6c2ec5e-5c42-458b-8ba5-7ce410fae44c)

**Comment by Navid200** [comment]
That means that you possibly selected the main version instead of the minimal version.  
Please make sure you have chosen the minimal release as shown in the screenshots or the video clips.

**Comment by Navid200** [comment]
So, you are correct that even if you pick the correct version, you still need to make sure you pick the minimal version.  Otherwise, the installation will take much more space.  We have wanted to minimize the footprint to minimize the load on the CPU and RAM as those are both quite limited for a weak machine that would satisfies the free tier requirements.

I hope this makes sense.
